zqd/middleware.go

```diff
@@ -78,7 +77,7 @@ func panicCatchMiddleware(logger *zap.Logger) mux.MiddlewareFunc {
 					return
 				}
 				rstr := fmt.Sprint(r)
-				logger.Error("panic", zap.String("error", rstr))
+				logger.DPanic("Panic", zap.Any("error", rstr))
```

Comment:
Does `zap.Any` do something special? `rstr` is a string, so wondering what the output difference will be.

Reply:
It was a comment from @nwt. But I think he meant I should get rid of the `fmt.Sprint(r)` and do `zap.Any`

Reply:
@mattnibs: That's what I had in mind.

Reply:
But actually the fact that we also return the string in the below `http.Error` says to me that we should keep it as-is.

Reply:
Right you are. Sorry I missed that.